### PR TITLE
Hot Fix: readVFAT3ADCLocal now uses cached registers [develop]

### PR DESCRIPTION
**Pull request by bdorney**
## Description

After CTP7 FW version [3.7.1](https://github.com/evka85/GEM_AMC/releases/tag/v3.7.1) the `ADCX` registers have been converted to `ADCX_CACHED` and `ADCX_UPDATE` for `X = 0,1`. 

This now makes use of these registers.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context

See discussion above.

## How Has This Been Tested?

Change was trivial.

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by jsturdy — CHANGES_REQUESTED** [review]
Hotfixes that go on `master` directly, come into `develop` from the same `hotfix` branch.
```
git co gemdaq/master
git co -b hofix/my-special-fix
git add <changes>
git ci -m "my hotfix that fixes bug X as listed in issue #Y
## make PR to master
## make PR to develop
## PR is merged to master with --no-ff
## PR is merged to develop with --ff (allows --ff-only if possible)
```

If you want to simulate locally
```
git co gemdaq/master
git co -b gemdaq/master/merge-test
git merge --no-ff hotfix/my-special-fix
###
git co gemdaq/develop
git co -b gemdaq/develop/merge-test
git merge --ff-only hotfix/my-special-fix
# if this fails, the fallback will be to create a merge commit, so if you want to *know* that the ff-only merge won't work, use two commands, otherwise, the below will try ff-only, and make a merge commit if it can't
git merge --ff hotfix/my-special-fix
```

TL;DR: This PR should be closed, and reopened from the branch being merged into `master`

**Comment by bdorney** [comment]
> TL;DR: This PR should be closed, and reopened from the branch being merged into `master`

Okay.  Will do that.
